lib/elixir/lib/calendar.ex

```diff
@@ -194,6 +234,16 @@ defmodule Date do
   end
 
   @doc """
+  Returns the number of days in a week.
+
+  Always returns 7 days in a week irrespective of the
+  calendar in use.
+  """
+  def days_in_a_week() do
```

Comment:
In case you are reading my replies from e-mail, please ignore the lower half of my previous reply. It was there by accident. :)